### PR TITLE
Major updates to instruction flow

### PR DESCRIPTION
**Pull request by AngieRudduck**
# Module: 00
## Lab/Demo: 00

Fixes # .

Changes proposed in this pull request:

-
-
-

### CODE REVIEW

**Review by AngieRudduck — COMMENTED** [review]
Minor updates made based on review.

File: Instructions/06-design-report-in-power-bi-desktop.md [hunk 1]
```diff
@@ -9,579 +9,400 @@ lab:
...
 
-3. To add an image, on the **Insert** ribbon tab, from inside the **Elements** group, click **Image**.
+1. In the **Fields** pane, drag the **Date \| Year** field (not the **Year** level of the hierarchy) into the slicer **Field** in Visualizations pane.
+    1. *The labs use a shorthand notation to reference a field. It will look like this: **Date \| Year**. In this example, **Date** is the table name and **Year** is the field name.*
 
-	![Picture 1](Linked_image_Files/07-design-report-in-power-bi-desktop_image15.png)
+1. To convert the slicer from a list to a dropdown, at the top-right of the slicer, select the down-arrow, and then select **Dropdown**.
 
-4. In the **Open** window, navigate to the **D:\PL300\Resources** folder.
+    ![Slicer style](Linked_image_Files\06_slicer_style.png)
```
Comment (Mihai-Ac) [inline Instructions/06-design-report-in-power-bi-desktop.md:80]: Photo is missing
image

File: Instructions/06-design-report-in-power-bi-desktop.md [hunk 2]
```diff
@@ -9,579 +9,400 @@ lab:
...
+     ![Picture 52](Linked_image_Files/07-design-report-in-power-bi-desktop_image30.png)
 
-	![Picture 51](Linked_image_Files/07-design-report-in-power-bi-desktop_image26.png)
+1. De-select the chart by clicking an empty area of the report page.
 
-23. Resize and position the visual so it sits to the right of the logo, and so it fills the width of the report page.
+1. To add a chart to the page, in the **Visualizations** pane, select the **Stacked Column Chart** visual type.
 
-	![Picture 26](Linked_image_Files/07-design-report-in-power-bi-desktop_image27.png)
+     ![Picture 53](Linked_image_Files/07-stacked-column-chart.png)
```
Comment (Mihai-Ac) [inline Instructions/06-design-report-in-power-bi-desktop.md:138]: Missing image here
image

File: Instructions/06-design-report-in-power-bi-desktop.md [hunk 3]
```diff
@@ -9,579 +9,400 @@ lab:
...
-6. In the column/line chart, select any month column to cross filter the page.
-
-7. While pressing the **Ctrl** key, select an additional month.
+1. In the Microsoft Edge browser window, in the Power BI service, in the **Navigation** pane (located at the left, and it could be collapsed), expand **My Workspace**.
 
-	*By default, cross filtering filters all other visuals on the page.*
+1. Open a Microsoft Edge browser, then sign-in at **https://app.powerbi.com**.
 
-8. Notice that the bar chart is filtered and highlighted, with the bold portion of the bars representing the filtered months.
+    ![Picture 93](Linked_image_Files/06-my-workspace-new.png)
```
Comment (Mihai-Ac) [inline Instructions/06-design-report-in-power-bi-desktop.md:338]: Missing image
image

File: Instructions/07-design-report-in-power-bi-desktop-enhanced.md [hunk 4]
```diff
@@ -1,488 +1,378 @@
...
-5. If prompted by Microsoft Edge to stay signed in, click **Yes**.
+1. Complete the sign-in process with your organizational credentials (or those provided to you). If prompted by Microsoft Edge to stay signed in, select **Yes**.
 
-6. In the Microsoft Edge browser window, in the Power BI service, in the **Navigation** pane, expand **My Workspace**.
+1. In the Microsoft Edge browser window, in the Power BI service, in the **Navigation** pane, expand **My Workspace**. Leave the Microsoft Edge browser window open.
 
- 	![Picture 22](Linked_image_Files/08-design-report-in-power-bi-desktop-enhanced_image3.png)
-
-7. Leave the Microsoft Edge browser window open.
+     ![Picture 22](Linked_image_Files/07-my-workspace-new.png)
```
Comment (Mihai-Ac) [inline Instructions/07-design-report-in-power-bi-desktop-enhanced.md:58]: Missing picture
image

File: Instructions/08-perform-data-analysis-in-power-bi-desktop.md [hunk 5]
```diff
@@ -9,303 +9,201 @@ lab:
...
 
-1. If prompted by Microsoft Edge to stay signed in, click **Yes**.
+1. In the Microsoft Edge browser window, in the Power BI service, in the **Navigation** pane, expand **My Workspace**. Leave the Microsoft Edge browser window open.
 
-1. In the Microsoft Edge browser window, in the Power BI service, in the **Navigation** pane, expand **My Workspace**.
-
- 	![Picture 4](Linked_image_Files/10-perform-data-analysis-in-power-bi-desktop_image3.png)
-
-1. Leave the Microsoft Edge browser window open.
+     ![Picture 22](Linked_image_Files/07-my-workspace-new.png)
```
Comment (Mihai-Ac) [inline Instructions/08-perform-data-analysis-in-power-bi-desktop.md:56]: Missing photo
image

File: Instructions/09-create-power-bi-dashboard.md [hunk 6]
```diff
@@ -9,360 +9,240 @@ lab:
...
+1. In the Microsoft Edge browser window, in the Power BI service, in the **Navigation** pane, expand **My Workspace**. Leave the Microsoft Edge browser window open.
 
-7. If prompted by Microsoft Edge to stay signed in, click **Yes**.
-
-8. In the Microsoft Edge browser window, in the Power BI service, in the **Navigation** pane, expand **My Workspace**.
-
-    ![Picture 40](Linked_image_Files/09-create-power-bi-dashboard_image4.png)
-
-9. Leave the Microsoft Edge browser window open.
+     ![Picture 22](Linked_image_Files/07-my-workspace-new.png)
```
Comment (Mihai-Ac) [inline Instructions/09-create-power-bi-dashboard.md:58]: Missing image
image

**Review by Mihai-Ac — CHANGES_REQUESTED** [review]
- Checked all labs.

- did not open .pbix files

- Created new commit for each lab with fixes for numbered comments/tips (left comments on each commit).

- Found 4 labs where images were missing (review comments left in the files. They are :

_Instructions/06-design-report-in-power-bi-desktop.md_ : 

**Slicer style](Linked_image_Files\06_slicer_style.png)
Picture 53](Linked_image_Files/07-stacked-column-chart.png)
Picture 93](Linked_image_Files/06-my-workspace-new.png**

For 
_07-design-report-in-power-bi-desktop-enhanced.md_
and
_08-perform-data-analysis-in-power-bi-desktop.md_
and
_09-create-power-bi-dashboard.md_

One image was missing: **Picture 22](Linked_image_Files/07-my-workspace-new.png)**

These images will need to be added, but everything else seems fine.

**Pull request merged**